README.md

```diff
@@ -114,29 +114,26 @@ spark.sql.catalog.tidb_catalog.pd.addresses  ${your_pd_adress}
 ```
 ./bin/spark-shell --jars /wherever-it-is/tispark-${name_with_version}.jar
 ```
-
-### Use TiSpark
-For TiSpark version 2.4.x:
+### Get TiDB Version
```

Comment:
<img width="421" alt="image" src="https://user-images.githubusercontent.com/17768378/160594999-0a659585-ff8f-438c-9f5f-8adc4d10a9b2.png">
It’s TiSpark's version